src/conda_package_handling/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "2.0.0a4"
+__version__ = "2.0.0b2"
```

Comment:
rever appears to have a cache that prevents b1 from being released twice, even after undoing the commit

Reply:
Did work as expected in a new clone, allowing me to try running rever again for a particular version number.

Reply:
this is why the `--force` argument is included in the release docs